<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationGeneral</class>
 <widget class="QWidget" name="ConfigurationGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="layoutgeneral">
     <property name="title">
      <string>Global Preferences</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="AlwaysUseDefaults">
        <property name="toolTip">
         <string>&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;(Recommended)&lt;/span&gt; This option requires RoboJournal to suggest (but not force) default settings wherever applicable.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Always suggest &amp;default settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="UseConfirmation">
        <property name="toolTip">
         <string>&lt;p&gt;&lt;b&gt;(Recommended)&lt;/b&gt; This safety protocol requires RoboJournal to require user confirmation before executing commands that delete data. &lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Always show &amp;confirmation dialogs</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="UseToolbarLabels">
        <property name="toolTip">
         <string>&lt;p&gt;Requires RoboJournal to display text labels on the toolbar buttons (as opposed to icon-only). Text labels may be cumbersome if the toolbar is docked left or right so some users may wish to turn them off.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Show &amp;labels on toolbar buttons</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="NameTitlebar">
        <property name="toolTip">
         <string>&lt;p&gt;If checked, RoboJournal displays the name of the current journal in the titlebar &lt;nobr&gt;(i.e. &lt;i&gt;journal_name&lt;/i&gt; - RoboJournal).&lt;/nobr&gt;&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Display current journal &amp;name in title bar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="Usealternating">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;If checked, RoboJournal alternates the row background colors in the Chronological Entry List and Search list. This helps to provide visual contrast in long lists but may look bad with some color schemes (so you may wish to disable it in such instances).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Display &amp;alternating row colors in lists</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="TagReminder">
        <property name="toolTip">
         <string>&lt;p&gt;This housekeeping function causes RoboJournal to display a list of all untagged entries it finds during every login (essentially a &amp;quot;nag window&amp;quot;). This feature can be helpful but some users may wish to disable it. &lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Display Tag &amp;Reminder window during login</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Toolbar docking position</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ToolbarPosition">
          <item>
           <property name="text">
            <string>Left</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Top (default)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="layoutdate">
     <property name="title">
      <string>Date and Time</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="DOW">
        <property name="toolTip">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans'; font-size:10pt;&quot;&gt;If checked, this setting displays the day of week when an entry was written in addition to the regular date and time. (i.e. Saturday, 1/1/2000).&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Display day of &amp;week when entries were made</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="RecordTime">
        <property name="toolTip">
         <string>&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;(Recommended) &lt;/span&gt;This causes new entries to be marked with the timestamp of when they were created. Entries without a timestamp are marked with &lt;span style=&quot; font-style:italic;&quot;&gt;[unknown time]&lt;/span&gt;.&lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Record the &amp;time when entries are made</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="UseMilitary">
        <property name="toolTip">
         <string>&lt;p&gt;Use this option if you wish to display timestamps in 24 hour format (i.e. 16:00 instead of 4:00 pm). &lt;/p&gt;</string>
        </property>
        <property name="text">
         <string>Use 24-&amp;hour clock in timestamps</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Preferred date format</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="DateFormat">
          <item>
           <property name="text">
            <string>European</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>USA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ISO-8601</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>6</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="DateExample">
        <property name="text">
         <string>Example: January 1, 2012 (MM/DD/YYYY)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Backend (Database)</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>RoboJournal should use</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Backend">
          <property name="toolTip">
           <string>This menu selects the type of database RoboJournal should currently work with. Only one backend may be used at a time.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
